Src/AutoFixture/EnumerableExtensions.cs

```diff
@@ -20,17 +27,17 @@ internal static int IndexOf<T>(this IEnumerable<T> items, T item)
             return -1;
         }
 
-        internal static IEnumerable<T> Insert<T>(this IEnumerable<T> items, int index, T item)
+        public static IEnumerable<T> Insert<T>(this IEnumerable<T> items, int index, T item)
```

Comment:
You sure you wan't this public?

Reply:
Notice, the `EnumerableExtensions` class is internal, so all the methods are still internal, even though they are marked as `public`.